packages/css-data/bin/mdn-data.ts

```diff
@@ -49,6 +49,18 @@ const normalizedValues = {
   "text-size-adjust": autoValue,
 } as const;
 
+const beautifyKeyword = (property: string, keyword: string) => {
+  if (keyword === "currentcolor") {
+    return "currentColor";
+  }
+  // builder style panel cannot interpret "normal" and "bold"
+  // always expected numeric value
+  if (property === "font-weight" && keyword === "normal") {
+    return "400";
+  }
```

Comment:
Strange that comment says `"normal" and "bold"`, but in code we handle only `normal`

Reply:
You can find bold in html.ts too